### ISSUE TITLE
[BUG] Shift-click to move a stack of items in/out of a GTCE machine/chest is rarely failing

### ISSUE THREAD

**Issue by anpilley** [GregTechCE/GregTech#1460]
**Describe the bug**
Since I started testing a dev version of Omnifactory (snapshot e3e6711), I've been having an issue when interacting with GTCE machines that wasn't showing up on earlier versions of the pack (that have earlier builds of GTCE)

Shift-clicking a stack to move it into, or out of, a GTCE chest or machine/bus is rarely failing to complete successfully. This happens most often when shift-clicking multiple successive stacks (like 8 or so stacks of some kind of dust, for instance). The UI updates, initially, showing all the stack have been moved, but then a short period later, things sync back up, and one of the multiple stacks will not have been moved, and is in its original location.

This is happening in both directions, either into a GT inventory block, or out of a GT inventory block. 

This was not happening in earlier versions of the pack, and since it's fairly early game, there's no significant TPS lag (TPS is clamped at 20 with no noticeable lag spikes). This isn't happening with other mods containers (like AA crates or vanilla minecraft chests) in the same pack that I've noticed.

This is unfortunately relatively rare, it's only happening roughly every 30 minutes to an hour or so of normal play, so it's difficult to catch and reproduce, but it's happening often enough that it doesn't feel like I'm just not clicking or missing the stack with my mouse (and no other issues with the mouse are noticeable in other games, currently)

**Versions**
Forge: 14.23.5.2847
GTCE: 1.10.9.612
_Modpack:_ Omnifactory dev snapshot e3e6711 (https://github.com/OmnifactoryDevs/Omnifactory/releases/tag/latest-dev-preview). Unfortunately, that specific dev snapshot has been replaced with a newer one at this point, but I only picked up the dev snapshot a few days ago, so there shouldn't be significant changes.
_Addons:_ No extra addons.

**Setup**
Playing Solo, in void dimension (via void cake), standard default lost cities world gen for starting overworld.

**Steps To Reproduce**
1) Have chest with multiple stacks of items (3-8 stacks of 64 or so)
2) use shift-click to rapidly move all the stacks into/out of inventory into/out of GT bus/chest/machine. 
3) Observe that all of the stacks have been moved to player's inventory.
4) close block's inventory UI.
5) move to another container
6) open GT container, begin moving items in using shift-click
7) Notice that one of the stacks failed to make it across from the original starting location.

**Expected behavior**
All stacks moved as the UI showed in step 3, with no stacks unmoved. This behavior worked fine when using GTCE 1.8.4.419 (in Omnifactory 1.2.2)

**Screenshots**
I'll try to capture video of it happening at some point in the near future, if only to confirm that I'm correct about my observation of point 3).

**Comment by anpilley** [comment]
Caught a video of it in action. You can see how I visibly move 4 stacks out of the output bus, but when I get to the crafting table, one of the stacks is missing (it's still in the output bus, although i don't include that in the video). See [this 14 second clip](https://www.youtube.com/watch?v=dFf-frXxAB0) for an example of what I'm getting.

(Sorry about the weird video dimensions, video capture didn't handle the maximized window instance of minecraft too well.)

**Comment by Archengius** [comment]
Can confirm it's not exclusively the Omnifactory bug, had it happen to me multiple times while playing FTB: Interactions too. Although for me it happened a bit more frequently, but the behavior was completely the same as described one.

**Comment by anpilley** [comment]
Attaching another example going in the opposite direction. shift-click on items in inventory to put into GT chest, one of the stacks doesn't make it and shows back up in my inventory, despite initially appearing to get there.

https://user-images.githubusercontent.com/12467241/107843852-49216200-6d83-11eb-868d-63f123a8d907.mp4

**Comment by warjort** [comment]
Can you try disabling mouse tweaks in the modpack.

I have seen similar, but not quite the same behaviour where it intermitently starts dropping items on the ground when you click
them with this mod enabled.

Restarting the game fixes the issue, but I used to always disable mouse tweaks when I saw it in a modpack.

Otherwise we need some way to "reliably" reproduce it so it can be debugged.

**Comment by anpilley** [comment]
I can give it a shot, however I'll point out that Omnifactory 1.2.2 also had the exact same MouseTweaks version (2.10), and I did full playthroughs (all the way to the final creative vending upgrade) on that pack version without issue.

I'm reasonably certain I've restarted the game a few times with the issue happening between each instance, but I'll double-check tomorrow.

**Comment by warjort** [comment]
The purpose of disabling mouse tweaks is to see if it still happens without it.

**Comment by warjort** [comment]
Yes, my guess is that it won't fix the problem, but it is an easy test to eliminate one possible source of the issue.
